src/errors/test-run/templates.js

```diff
@@ -108,7 +108,7 @@ export default {
     `),
 
     [TYPE.actionStringArrayElementError]: err => markup(err, `
-        Elements of the <code>${err.argumentName}</code> argument are expected to be non-empty strings, but the element at index ${err.elementIndex} was ${err.actualValue}.
+        Elements of the <code>${err.argumentName}</code> argument are expected to be non-empty strings, but the element at index <code>${err.elementIndex}</code> was ${err.actualValue}.
```

Comment:
I think we should wrap `actualValue` here not index. BTW, it worth cheking if `actualValue` not wrapped elsewhere